src/components/new-request/business-lookup.vue

```diff
@@ -14,7 +14,7 @@
       hint="Search by name, incorporation or registration number of existing business"
       item-text="name"
       item-value="identifier"
-      label="Find an existing business"
+      :label="lookupLabel"
```

Comment:
Now the label is set depending on whether we're searching for Active or Historical businesses. Ditto line 37 where the error depends on whether we found active/historical businesses or not.